index.html

```diff
@@ -69,6 +69,161 @@ <h1 class="header_job">I’m a software developer</h1>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+          </section>
+        </div>
+      </section>
+
+      <section id="about">
+        <section class="about_top">
+          <img src="/images/about-top2.png" alt="about-top" class="about-top" />
+          <img
+            src="/images/about-left.jpg"
+            alt="about-bottom"
+            class="about-bottom"
+          />
+          <img src="/images/about-top.jpg" alt="about-top" class="about-top2" />
+          <img
+            src="/images/about-bottom.jpg"
+            alt="about-bottom"
+            class="about-bottom2"
+          />
```

Comment:
- [ ] These images are not displaying as you can see from this screenshot

![image](https://user-images.githubusercontent.com/26736582/205171649-2224b14d-8a36-4a09-b3dd-9f4a2d81ecc0.png)

I think it will be better to use a background since the image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.